src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,73 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * A task that stores the deadline.
+ */
+public class Deadline extends Task {
+    /**
+     * The date of the deadline.
+     */
+    private LocalDate date;
```

Comment:
hmmm you didnt do the documentation for the attributes in the other classes, why suddenly this?